src/make/cards.js

```diff
@@ -18,7 +18,7 @@ var setsToIgnore = ['TSB','ITP','CP1','CP2','CP3'];
 before()
 
 var types = ['core', 'expansion', 'commander', 'planechase', 'starter', 'un']
-var codes = ['EMA', 'MMA', 'VMA', 'CNS', 'TPR', 'MM2', 'EXP', 'MPS', 'CN2', 'MM3', 'MPS_AKH', "IMA"]
+var codes = ['EMA', 'MMA', 'VMA', 'CNS', 'TPR', 'MM2', 'EXP', 'MPS', 'CN2', 'MM3', 'MPS_AKH', "IMA", "BBD"]
```

Comment:
Wondering why masters sets are not filtered by `masters` type, but some are listed by their codes instead?!